en/docs/reference/mediators/about-mediators.md

```diff
@@ -38,26 +38,70 @@ Mediators are classified as follows based on whether or not they access the mess
-
-**Advanced Mediators**
-
-[Cache]({{base_path}}/reference/mediators/cache-mediator) | [ForEach]({{base_path}}/reference/mediators/foreach-mediator) | [Clone]({{base_path}}/reference/mediators/clone-mediator) | [Store]({{base_path}}/reference/mediators/store-mediator) | [Iterate]({{base_path}}/reference/mediators/iterate-mediator) | [Aggregate]({{base_path}}/reference/mediators/aggregate-mediator) | [Callout]({{base_path}}/reference/mediators/callout-mediator) | [Transaction]({{base_path}}/reference/mediators/transaction-mediator) | [Throttle]({{base_path}}/reference/mediators/throttle-mediator) | [DBReport]({{base_path}}/reference/mediators/db-report-mediator) | [DBLookup]({{base_path}}/reference/mediators/dblookup-mediator) | [EJB]({{base_path}}/reference/mediators/ejb-mediator) | [Binder]({{base_path}}/reference/mediators/builder-mediator) | [Entitlement]({{base_path}}/reference/mediators/call-mediator) | [OAuth]({{base_path}}/reference/mediators/call-mediator) | [Smooks]({{base_path}}/reference/mediators/smooks-mediator) | [Data Mapper]({{base_path}}/reference/mediators/data-mapper-mediator) | 
-
-**Extension Mediators**
-
-[Class]({{base_path}}/reference/mediators/class-mediator) | [Script]({{base_path}}/reference/mediators/script-mediator) |
+## List of mediators
+
+WSO2 Micro Integrator includes a comprehensive library of mediators that provide functionality for implementing widely used [Enterprise Integration Patterns (EIPs)]({{base_path}}/learn/enterprise-integration-patterns/eip-overview/). You can also easily write a custom mediator to provide additional functionality using various technologies such as Java, scripting, and Spring.
+
+### Flow control mediators
+
+- [Filter mediator]({{base_path}}/reference/mediators/filter-mediator)
+- [Switch mediator]({{base_path}}/reference/mediators/switch-mediator)
+- [Clone mediator]({{base_path}}/reference/mediators/clone-mediator)
+- [Iterate mediator]({{base_path}}/reference/mediators/iterate-mediator)
+- [ForEach mediator]({{base_path}}/reference/mediators/foreach-mediator)
+- [Aggregate mediator]({{base_path}}/reference/mediators/aggregate-mediator)
+- [Validate mediator]({{base_path}}/reference/mediators/validate-mediator)
+
+### Transformation mediators
+
+- [Enrich mediator]({{base_path}}/reference/mediators/enrich-mediator)
+- [Header mediator]({{base_path}}/reference/mediators/header-mediator)
+- [XSLT mediator]({{base_path}}/reference/mediators/xslt-mediator)
+- [JSON Transform mediator]({{base_path}}/reference/mediators/json-transform-mediator)
+- [Fault mediator]({{base_path}}/reference/mediators/fault-mediator)
+- [PayloadFactory mediator]({{base_path}}/reference/mediators/payloadfactory-mediator)
+- [Data Mapper mediator]({{base_path}}/reference/mediators/data-mapper-mediator)
+
+### Extension mediators
+
+- [Script mediator]({{base_path}}/reference/mediators/script-mediator)
+- [Class mediator]({{base_path}}/reference/mediators/class-mediator)
+
+### Generic mediators
```

Comment:
Shall we move this to top, as these are widely used.

Reply:
Fixed via https://github.com/wso2/docs-mi/pull/852/commits/386fc20e8778245e9abeb8dccd84e68ba2b67a6a.